internal/armada/repository/event_job_status.go

```diff
@@ -8,13 +8,53 @@ import (
 )
 
 type EventJobStatusProcessor struct {
-	stream        eventstream.EventStream
 	queue         string
 	jobRepository JobRepository
+	stream        eventstream.EventStream
+	batcher       eventstream.EventBatcher
 }
 
-func NewEventJobStatusProcessor(stream eventstream.EventStream, queue string, jobRepository JobRepository) *EventJobStatusProcessor {
-	return &EventJobStatusProcessor{stream: stream, queue: queue, jobRepository: jobRepository}
+func NewEventJobStatusProcessor(
+	queue string,
+	jobRepository JobRepository,
+	stream eventstream.EventStream,
+	batcher eventstream.EventBatcher,
+) *EventJobStatusProcessor {
+	batcher.Register(func(events []*api.EventMessage) error {
+		var jobStartInfos []*JobStartInfo
+		for _, eventMessage := range events {
+			event, err := api.UnwrapEvent(eventMessage)
+			if err != nil {
+				log.Errorf("error while unwrapping event message: %v", err)
+				return err
+			}
+			switch event := event.(type) {
+			case *api.JobRunningEvent:
+				jobStartInfos = append(jobStartInfos, &JobStartInfo{
+					JobId:     event.GetJobId(),
+					ClusterId: event.ClusterId,
+					StartTime: event.Created,
+				})
+			}
+		}
+
+		jobErrors, err := jobRepository.UpdateStartTime(jobStartInfos)
+		if err != nil {
+			log.Errorf("error when updating start times for jobs: %v", err)
+			return err
+		}
+		for _, err := range jobErrors {
+			if err != nil {
+				log.Errorf("error when updating start time for single job: %v", err)
```

Comment:
why doesn't this return an error?  Is it the case that the caller doesn't need to know what's gone wrong?